CMakeLists.txt

```diff
@@ -72,6 +72,14 @@ if(USE_R35)
 endif(USE_R35)
 
 if(BUILD_FOR_R)
+    list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake/modules")
+    find_package(LibR REQUIRED)
+    message(STATUS "LIBR_HOME: ${LIBR_HOME}")
+    message(STATUS "LIBR_EXECUTABLE: ${LIBR_EXECUTABLE}")
+    message(STATUS "LIBR_INCLUDE_DIRS: ${LIBR_INCLUDE_DIRS}")
+    message(STATUS "LIBR_LIB_DIR: ${LIBR_LIB_DIR}")
+    message(STATUS "LIBR_CORE_LIBRARY: ${LIBR_CORE_LIBRARY}")
```

Comment:
I think it's important to have `LIBR_EXECUTABLE` too. I have been using that a lot in debugging. Since we use that to find everything else, it's important to know which one was found.

I'd be ok with dropping  `LIBR_HOME` and `LIBR_LIB_DIR` though.

Reply:
Sure, agree with you!

Reply:
removed in https://github.com/microsoft/LightGBM/pull/2901/commits/20aeb82830f3c474fc4f2a55e4afedb06b5016b5